### ISSUE TITLE
[FEATURE REQUEST] Lightbox preview

### ISSUE THREAD

**Issue by didix16** [ebess/advanced-nova-media-library#240]
Hi there!
First of all thanks for your effort doing this great package! However I'm missing something that would be nice. I've a field which is multiple image but when I gonna preview it, it always open in a new browser window instead of showing some kind of modal like lightbox to view the gallery.

So how about using it to preview multiple images, using arrow controls to view the next one?

Thanks in advance!

**Comment by basserd** [comment]
It's been a few years, have you implemented this somehow? @didix16

**Comment by didix16** [comment]
Hi @basserd. Sorry I have not implemented any change. I'm using the packge without the improvements. Hope they apply anytime

**Comment by basserd** [comment]
@didix16 I've implemented a Javascript method that injects the use of photoswipe for the detail view of the resource. I know it's not the best solution. But it was something that I could quickly come up with.

**Comment by didix16** [comment]
So you did a custom nova tool that loads your JavaScript? If so, could you share the code? Thanks in advance!

**Comment by basserd** [comment]
Well it could be changed to be within a tool. But I just loaded it from within a javascript file from the resources folder.

**Comment by didix16** [comment]
Yeah I think it could be  turned into a tool. At the moment I'll try your solution and see how it works. I'll speak with my front-end partner to propperly inject into our project.

Thank you so much @basserd !
